docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory (`./`) to `/app` in the container can lead to potential issues, such as overwriting files in the container or including unnecessary files. Consider specifying only the necessary directories or files to be mounted.